### PR TITLE
OSDOCS-5509 Update Customer Cloud Subscription AWS Accounts

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-5509](https://issues.redhat.com//browse/OSDOCS-5509)

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OSDOCS-5509

Link to docs preview:
OSD: https://57079--docspreview.netlify.app/openshift-dedicated/latest/osd_planning/aws-ccs.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jneczypor** [comment]
Hi @xueli181114! Can you please review this when you have a moment. Thank you!

**Review by arendej — APPROVED** [review]
LGTM

File: modules/ccs-aws-iam.adoc [hunk 1]
```diff
@@ -107,7 +107,11 @@ The `osdManagedAdmin` user is created immediately after taking control of the cu
 
 ** AmazonEC2ReadOnlyAccess
 ** CustomerAdministratorAccess
-
++
+[NOTE]
+====
+The `network-mgmt` role only applies to non-Customer Cloud Subscription (CCS) clusters.
+====
```
Comment (bscott-rh) [inline modules/ccs-aws-iam.adoc:114]: The writing looks good to me, but I don't think this needs to be in a [NOTE]. What do you think about adding it to line 106 before the sentence "The following policies are attached to the role:"? The way it's formatted now, it looks like the note only applies to the `CustomerAdministratorAccess` role.

**Comment by bscott-rh** [comment]
Added one piece of feedback. Also, please squash down to 1 commit.

File: modules/ccs-aws-iam.adoc (hunk 1 above)
Comment (jneczypor) [reply modules/ccs-aws-iam.adoc]: Thanks so much! I made the changes. Please let me know if there is anything else you find.

**Comment by jneczypor** [comment]
Thanks @bscott-rh! I think it should be ready to go now.

**Comment by bscott-rh** [comment]
LGTM!

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Review by skrthomas — COMMENTED** [review]
Merge review lgtm

**Pull request merged**